imports/plugins/included/payments-stripe-connect/client/settings/settings.js

```diff
@@ -0,0 +1,43 @@
+import { Template } from "meteor/templating";
+import { Reaction, i18next } from "/client/api";
+import { Packages } from "/lib/collections";
+import { StripeConnectPackageConfig } from "../../lib/collections/schemas";
+
+import "./settings.html";
+
+Template.stripeConnectSettings.helpers({
+  StripeConnectPackageConfig() {
+    return StripeConnectPackageConfig;
+  },
+  packageData() {
+    return Packages.findOne({
+      name: "reaction-stripe-connect",
+      shopId: Reaction.getShopId()
+    });
+  }
+});
+
+AutoForm.hooks({
+  "stripe-connect-update-form": {
+    onSuccess: function () {
+      return Alerts.toast(i18next.t("admin.settings.saveSuccess"), "success");
+    },
+    onError: function () {
+      return Alerts.toast(`${i18next.t("admin.settings.saveFailed")} ${error}`, "error");
+    }
+  }
+});
+
+Template.stripeConnectRedirect.onCreated(function () {
+  // TODO: Verify that this works and define steps to reproduce.
+  // grab stripe connects oauth values and redirect the user
+  const authCode = FlowRouter.getQueryParam("code");
```

Comment:
I think this should be Reaction.Router -> directly using flow router could be an issue during upgrades.